src/main/java/ch/njol/skript/effects/EffToggle.java

```diff
@@ -123,70 +132,80 @@ public boolean init(final Expression<?>[] vars, final int matchedPattern, final
 	
 	@Override
 	protected void execute(final Event e) {
```

Comment:
Since you're editing, remove all `final` modifier of method parameters  \:)
```suggestion
	protected void execute(Event e) {
```